packages/documentation/pages/foundations/color.vue

```diff
@@ -149,3 +149,22 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
```

Comment:
Why is this copy-pasted here?

Reply:
This component is also using the styles that are now scoped to KtHeading.

Reply:
Why does this component not use KtHeading then?